portafly/portafly_api.yml

```diff
@@ -9,45 +9,44 @@ info:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0
 paths:
-  /developer/accounts:
-    summary: A list of developer accounts
-    description: A list of developer accounts.
+  /users:
+    summary: Users
+    description: ""
```

Comment:
I guess the idea is to not have a `description` for all possible operations, but a description for every single one, like defined in line 18
Then we can remove this, instead of keeping an empty string